drake-torch.dockerfile

```diff
@@ -115,6 +154,10 @@ RUN apt-get update && apt-get install -y \
     iputils-ping \
     && rm -rf /var/lib/apt/lists/*
 
+# plase setup.bash script into bashrc file
+RUN touch $HOME/.bashrc \
+ && echo '/opt/ros/${ROS_DISTRO}/setup.bash' > $HOME/.bashrc
+
```

Comment:
Shouldn't this go in `scripts/fix_bashrc.sh`?

Reply:
fixed